lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.scss

```diff
@@ -0,0 +1,47 @@
+adf-search-chip-autocomplete-input {
+    .adf-chip-list {
+        width: 100%;
+    }
+
+    .mat-form-field-appearance-outline .mat-form-field-outline {
+        top: 0;
+    }
+
+    .mat-form-field-infix {
+        border: none;
+    }
+
+    .adf-option-chips {
+        border: 1px solid var(--theme-text-color);
+        border-radius: 10px;
+        color: var(--theme-primary-color-default-contrast);
+        background-color: var(--theme-primary-color-default-contrast) !important;
+        height: auto;
+        word-break: break-word;
+    }
+
+    .adf-option-chips-delete-icon {
+        font-size: 13px !important;
```

Comment:
I see you use many !important in this file. Can you check if there is a way to avoid using them? You have encapsulation set to None so you should be able to override styles even from child components.

Reply:
checked, not applied without `!important `